src/components/Scheduler/AvailableTimes.tsx

```diff
@@ -0,0 +1,41 @@
+import { useSchedulerContext } from '@/hooks/useSchedulerContext';
+import { MeetingSchedule } from '@/types/meetings';
+import classNames from 'classnames';
+import { format } from 'date-fns';
+import { Button } from '../common/Button';
+
+interface Props {
+  selectedDay: MeetingSchedule;
+  dateFormat?: 'hh:mm a' | 'HH:mm';
+}
+
+export const AvailableTimes = ({
+  selectedDay,
+  dateFormat = 'hh:mm a',
+}: Props) => {
+  const { setSelectedHour, selectedHour } = useSchedulerContext();
+
+  return (
+    <div className='flex flex-col items-center gap-5'>
+      <p>{format(selectedDay.day, 'EEEE, MMM dd')}</p>
```

Comment:
What does EEEE stand for in the date format?

Reply:
thats for the day of the week, you can found more format references here: [date-fns formats](https://date-fns.org/v3.3.1/docs/format)

Reply:
Got it!